Countries/CountryTableViewCell.swift

```diff
@@ -23,9 +23,9 @@ class CountryTableViewCell: UITableViewCell {
     static let identifier = "CountryTableViewCell"
     
     private let mainView: UIView = {
-        let bossView = UIView()
-        bossView.translatesAutoresizingMaskIntoConstraints = false
-        return bossView
+        let mainView = UIView()
+        mainView.translatesAutoresizingMaskIntoConstraints = false
+        return mainView
```

Comment:
this naming visible only inside closure, so can be just `view`.
 as for `mainView`, suggestion -> `containerView` or `cellContainerView`

Reply:
Ok)